docker-compose.yaml

```diff
@@ -14,9 +14,18 @@ services:
     environment:
       - SPARK_MODE=master
       - SPARK_MASTER_WEBUI_PORT=8090
+      - MAX_EXECUTORS=4
+      - POSTGRES_USER=hive
+      - POSTGRES_PASSWORD=hivepassword
+      - POSTGRES_DB=hive
+      - POSTGRES_URL=postgres:5432
+    volumes:
+      - ./cdr/cdm/jupyter:/cdm_shared_workspace
```

Comment:
Why do the master and workers need the shard space now?

Reply:
This is another thing I might not fully understand. But as far as I read it, hive will write to `warehouse.dir`. I am not sure which service is writing to it. But even though, master and workers now have the shared space, the `warehouse.dir` is still empty. It doesn't prevent users to read/query tables. I am guess this setting won't hurt.

Reply:
Maybe we should have a ticket to look into this later?

Reply:
yea. created an issue https://github.com/kbase/cdm-jupyterhub/issues/28. I anticipate that once we set up a standalone Hive service, this issue will no longer be relevant.

Reply:
:+1: